_opensearch/install/plugins.md

```diff
@@ -54,38 +54,18 @@ bin/opensearch-plugin list
   <tr>
     <td>1.1.0</td>
     <td>
-      <pre>analysis-icu                         1.1.0
-analysis-kuromoji                    1.1.0
-analysis-nori                        1.1.0
-analysis-phonetic                    1.1.0
-analysis-smartcn                     1.1.0
-analysis-stempel                     1.1.0
-analysis-ukrainian                   1.1.0
-discovery-azure-classic              1.1.0
-discovery-ec2                        1.1.0
-discovery-gce                        1.1.0
-ingest-attachment                    1.1.0
-mapper-annotated-text                1.1.0
-mapper-murmur3                       1.1.0
-mapper-size                          1.1.0
-opensearch-alerting                  1.2.0.0
+      <pre>opensearch-alerting                  1.2.0.0
```

Comment:
Should this have a `-` like line 57?

Reply:
That's from GitHub, actually.

Reply:
I realized that as I came back from lunch :D